athena-mysql/src/test/java/com/amazonaws/athena/connectors/mysql/MySqlQueryStringBuilderTest.java

```diff
@@ -105,4 +113,28 @@ public void generateSqlIsNotEqual() throws Exception
         assertEquals(expectedPreparedStatement, preparedStatement);
 
     }
+
+    @Test
+    public void testDatePredicate() throws Exception
```

Comment:
We need a test that validates that the long value of the date gets converted to the human readable date correctly. EG: 10440 converts to aug 2nd 1998